packages/composites/outlier-detection/schemas/env.json

```diff
@@ -44,7 +44,7 @@
           "required": ["DERIBIT_ADAPTER_URL"]
         },
         {
-          "required": ["OILPRICEAPI_COM_ADAPTER_URL"]
+          "required": ["OILPRICEAPI_ADAPTER_URL"]
         },
```

Comment:
Is this introducing a breaking change by changing this config var name?

Reply:
Should this PR be accompanied by a changeset?

Reply:
I don't think a changeset is needed. We are only fixing an error in the env schema file. The env var itself never changed, so there are no breaking changes (or code changes for that matter).